src/components/Confirm/index.js

```diff
@@ -147,7 +150,7 @@ class Confirm extends Component {
       errorKey = 'SERVER_ERROR'
     }
 
-    this.setState({ uploadInProgress: false })
+    this.setState({ uploadInProgress: false, btnDisabled: false })
```

Comment:
`btnDisabled` is redundant in this component, it's always the same as `uploadInProgress` so it should be removed

Reply:
Removed redundant state